radicle-cli/tests/commands.rs

```diff
@@ -394,6 +394,91 @@ fn rad_init_sync_and_clone() {
     .unwrap();
 }
 
+#[test]
+fn rad_fetch() {
+    let mut environment = Environment::new();
+    let working = environment.tmp().join("working");
+    let alice = environment.node("alice");
+    let bob = environment.node("bob");
+
+    let mut alice = alice.spawn(Config::default());
+    let bob = bob.spawn(Config::default());
+
+    alice.connect(&bob);
+    fixtures::repository(working.join("alice"));
+
+    // Necessary for now, if we don't want the new inventry announcement to be considered stale
+    // for Bob.
+    // TODO: Find a way to advance internal clocks instead.
+    thread::sleep(time::Duration::from_millis(3));
+
+    // Alice initializes a repo after her node has started, and after bob has connected to it.
+    test(
+        "examples/rad-init-sync.md",
+        &working.join("alice"),
+        Some(&alice.home),
+        [],
+    )
+    .unwrap();
+
+    // Wait for bob to get any updates to the routing table.
+    bob.converge([&alice]);
+
+    test(
+        "examples/rad-fetch.md",
+        working.join("bob"),
+        Some(&bob.home),
+        [],
+    )
+    .unwrap();
+}
+
+#[test]
+fn rad_fork() {
+    let mut environment = Environment::new();
+    let working = environment.tmp().join("working");
+    let alice = environment.node("alice");
+    let bob = environment.node("bob");
+
+    let mut alice = alice.spawn(Config::default());
+    let bob = bob.spawn(Config::default());
+
+    alice.connect(&bob);
+    fixtures::repository(working.join("alice"));
+
+    // Necessary for now, if we don't want the new inventry announcement to be considered stale
+    // for Bob.
+    // TODO: Find a way to advance internal clocks instead.
+    thread::sleep(time::Duration::from_millis(3));
```

Comment:
Do we need these sleeps in here? afaict there's only one inventory announcement.

Reply:
Turns out no it's not needed :)